apps/web/src/modules/job/components/Card/components/Header/index.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { mock } from '@modela/dtos'
+import { render } from '@testing-library/react'
+import { mockAndSpy } from 'common/utils/testing'
+import React from 'react'
+
+describe('<Header />', () => {
+  const ImageSpy = mockAndSpy('next/image')
+  const MOCK_TITLE = mock('job').get().title
+  const MOCK_COMPANY_NAME = mock('casting').get().companyName
+  const MOCK_CASTING_IMAGE = mock('user').get().profileImageUrl || ''
```

Comment:
```suggestion
  const MOCK_CASTING_IMAGE = mock('user').get().profileImageUrl
```
since profileImageUrl won't be null

Reply:
I got error saying it's string|null, look like it's from prisma shared type?